### ISSUE TITLE
help needed

### ISSUE THREAD

**Issue by NukeWilliams** [nuta/kerla#110]
hey @nuta
is there any way to b build Kerla without docker??

**Comment by nuta** [comment]
No. It's essential to build Linux binaries in a clean build environment.
